deliver/lib/deliver/options.rb

```diff
@@ -6,6 +6,7 @@ class Options
     def self.available_options
       user = CredentialsManager::AppfileConfig.try_fetch_value(:itunes_connect_id)
       user ||= CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
+      user ||= ENV["DELIVER_USER"]
```

Comment:
We already support `env_name: "DELIVER_USERNAME"` 3 lines below, seems like it's ben inconsistent so far